src/pages/register/client.vue

```diff
@@ -194,7 +198,22 @@ async function createAcc() : Promise<void> {
     }));
 
     // Autenticar al cliente
-    await router.push('/login');
+    if (response.value?.status == 201) {
+      auth.authenticate(UserCreated.value);
+      await router.replace('/l');
```

Comment:
Creo que esto se ha escrito mal
```suggestion
      await router.replace('/');
```

Reply:
Bien visto, gracias!